Dizzy/Screens/Discovery/DiscoveryVM.swift

```diff
@@ -0,0 +1,107 @@
+//
+//  DiscoveryViewModel.swift
+//  Dizzy
+//
+//  Created by Or Menashe on 01/04/2019.
+//  Copyright © 2019 Dizzy. All rights reserved.
+//
+
+import UIKit
+
+protocol DiscoveryViewModelType {
+    func numberOfSections() -> Int
+    func numberOfItemsForSection(_ section: Int) -> Int
+    func itemForIndexPath(_ indexPath: IndexPath) -> PlaceInfo
+    var navigationDelegate: DiscoveryViewModelNavigationDelegate? { get set }
+    var delegate: DiscoveryViewModelDelegate? { get set }
+    var currentLocation: Location? { get }
+    var currentCity: Observable<String> { get }
+    
+    func mapButtonPressed()
+}
+
+protocol DiscoveryViewModelDelegate: class {
+    func reloadData()
```

Comment:
like I said previously I think that the data should be observable and when the data update you reload it on the viewController

Reply:
love you